sql/hive/src/main/scala/org/apache/spark/sql/hive/HiveUtils.scala

```diff
@@ -404,6 +405,8 @@ private[spark] object HiveUtils extends Logging {
     propMap.put(ConfVars.METASTORE_EVENT_LISTENERS.varname, "")
     propMap.put(ConfVars.METASTORE_END_FUNCTION_LISTENERS.varname, "")
 
+    SparkHadoopUtil.get.appendSparkHadoopConfigs(propMap)
```

Comment:
We are unable to know this is getting the values from `sys.props`. How about changing the interface to?
```
// xyz
SparkHadoopUtil.get.appendSparkHadoopConfigs(sys.props.toMap, propMap)
```